src/pages/home/wallet/UnclaimedRoyalties.tsx

```diff
@@ -0,0 +1,63 @@
+import { Box, Button } from "@mui/material";
+import { Typography } from "elements";
+import theme from "theme";
+
+interface UnclaimedRoyaltiesProps {
+  unclaimedRoyalties: number;
+}
+
+export const UnclaimedRoyalties = ({
+  unclaimedRoyalties,
+}: UnclaimedRoyaltiesProps) => {
+  return (
+    <Box
+      sx={ {
+        backgroundColor: theme.colors.grey600,
+        padding: 2.5,
+        maxWidth: "400px",
+        height: "100px",
```

Comment:
I wonder if we should be setting a specific height here, i am afraid there could be an issue with the inner content being bigger than 100px

Reply:
Changed to minHeight